docs/man/git-lfs-clone.1.ronn

```diff
@@ -10,6 +10,10 @@ git-lfs-clone(1) -- Efficiently clone a LFS-enabled repository
 Clone an LFS enabled Git repository more efficiently by disabling LFS during the
 git clone, then performing a 'git lfs pull' directly afterwards.
 
+'git lfs clone' also installs all of the repo-level hooks (.git/hooks) that LFS
+requires to operate. If `--separate-git-dir` is given, the hooks will be
```

Comment:
FYI note for future-me: No code change was needed to support `--separate-git-dir`. One of the tests below calls `assert_hooks` though :+1: